File: Python/UberVan.py
from car import Car

class UberVan(Car):
    """Class UberVan instance, herency"""
    typeCarAccepted: list
    seatsMaterial: list

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

    # Override method setPassenger of super class Car for apply Polyomorphs
    def set_passenger(self, passenger):
        if passenger == 6:
            self.set_passenger = passenger
        else: 
            print("Minimal passenger acepted are 6.")

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car{
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    
    public UberVan(String license, Account driver){
        super(license, driver);
    }

    // public UberVan(String license, Account driver, 
    //     Map<String, Map<String,Integer>> typeCarAccepted,
    //     ArrayList<String> seatsMaterial){
    //         super(license, driver);
    //         this.seatsMaterial = seatsMaterial;
    //         this.typeCarAccepted = typeCarAccepted;
    //     }
    
    @Override
    public void setPassenger(Integer passenger){
        if (passenger >= 6){
            this.passenger = passenger;
        }else{
            extracted(passenger);
        }
    }

    private void extracted(Integer passenger) {
        System.out.println("Minimum passenger are 6 and you give " + passenger);
    }
}